src/Events/DispatchAudit.php

```diff
@@ -71,8 +71,13 @@ public function __serialize()
      */
     public function __unserialize(array $values): void
     {
-        $this->model = new $values['class'];
+        $model = new $values['class'];
 
+        if(! $model instanceof Auditable) {
```

Comment:
```suggestion
        if (! $model instanceof Auditable) {
```

Reply:
Pint will take care of this, but this must be merged first